server/tests/controller/article.spec.js

```diff
@@ -60,16 +61,20 @@ describe('Create an Article', async () => {
   });
 
   it('should return server error for create article', async () => {
-    const req = { body: {}, user: { userId: 'f1e2e1c8-6a90-4bcd-b349-49f989d311d6' } };
-    const res = {
-      status() {},
-      json() {}
-    };
-
-    sinon.stub(res, 'status').returnsThis();
-    sinon.stub(Article, 'create').throws();
-    await createArticle(req, res);
-    expect(res.status).to.have.been.calledWith(500);
+    try {
+      const req = { body: {}, user: { userId: 'f1e2e1c8-6a90-4bcd-b349-49f989d311d6' } };
+      const res = {
+        status() {},
+        json() {}
+      };
+
+      sinon.stub(res, 'status').returnsThis();
+      sinon.stub(Article, 'create').throws();
+      await createArticle(req, res);
+      expect(res.status).to.have.been.calledWith(500);
```

Comment:
you might want to remove  assertion from the `try-block`

Reply:
you should also consider using `..).to.be.equal(500)` to be consistent

Reply:
Thank you @Eazybee , There is a call to response body, thus the assertions can't be outside the try block. 
Then because that is a Sinon call to mock server, thus calledWith, for your reference https://www.chaijs.com/plugins/sinon-chai/.